QModManager/DataStructures/WeightedList.cs

```diff
@@ -1,4 +1,5 @@
-﻿namespace QModManager.DataStructures
+﻿#pragma warning disable CS1591 // Missing XML comment for publicly visible type or member
+namespace QModManager.DataStructures
```

Comment:
All classes of the `QModManager.DataStructures` namespace can be made `internal`.

This will resolve the compiler warnings about lack of documentation.